web/oauth.go

```diff
@@ -273,34 +273,45 @@ func completeOAuth(c *Context, w http.ResponseWriter, r *http.Request) {
 	body, teamId, props, tokenUser, err := c.App.AuthorizeOAuthUser(w, r, service, code, state, uri)
 
 	action := ""
+	hasRedirectUrl := false
+	isActionMobileAuth := false
+	redirectUrl := ""
 	if props != nil {
 		action = props["action"]
+		isActionMobileAuth = action == model.OAUTH_ACTION_MOBILE
+		if _, ok := props["redirect_to"]; ok {
+			redirectUrl = props["redirect_to"]
```

Comment:
You can use the first parameter of the lookup instead of using the blank identifier (`_`) and then you don't have to look it up twice.